requirements_dev_optional.txt

```diff
@@ -18,6 +18,7 @@ pytest-cov==2.7.1
 pytest-doctestplus==0.4.0
 pytest-remotedata==0.3.2
 h5py==2.10.0
-s3fs==0.5.0; python_version > '3.6'
+s3fs==0.5.1; python_version > '3.6'
+git+https://github.com/intake/filesystem_spec; python_version > '3.6'
```

Comment:
Is there a package on PyPI or conda-forge that we could install?

Reply:
Not yet, no.

Reply:
Do we have a rough sense when these might be created?

Reply:
When everything is good here, I can be sure to make a release before merging, so that this line can be reverted

Reply:
Sounds good @martindurant.